src/_gettsim/taxes/zu_verst_eink/vorsorgeaufw.py

```diff
@@ -273,69 +273,69 @@ def vorsorgeaufw_y_tu_bis_2004(
     """
     multiplikator1 = max(
         (
-            12 * (ges_rentenv_beitr_m_tu + ges_krankenv_beitr_m_tu)
-            - _vorsorgeaufw_vom_lohn_y_tu_bis_2004
+            12 * (ges_rentenv_beitr_m_sn + ges_krankenv_beitr_m_sn)
+            - _vorsorgeaufw_vom_lohn_y_sn_bis_2004
         ),
         0.0,
     )
 
-    item_1 = (1 / anz_erwachsene_tu) * multiplikator1
+    item_1 = (1 / anz_personen_sn) * multiplikator1
 
     if item_1 > eink_st_abzuege_params["vorsorge_2004_grundhöchstbetrag"]:
         multiplikator2 = eink_st_abzuege_params["vorsorge_2004_grundhöchstbetrag"]
     else:
         multiplikator2 = item_1
 
-    item_2 = (1 / anz_erwachsene_tu) * multiplikator2
+    item_2 = (1 / anz_personen_sn) * multiplikator2
 
     hoechstgrenze_item3 = (
-        anz_erwachsene_tu * eink_st_abzuege_params["vorsorge_2004_grundhöchstbetrag"]
+        anz_personen_sn * eink_st_abzuege_params["vorsorge_2004_grundhöchstbetrag"]
     )
 
     if (item_1 - item_2) > hoechstgrenze_item3:
         item_3 = 0.5 * hoechstgrenze_item3
     else:
         item_3 = 0.5 * (item_1 - item_2)
 
-    out = _vorsorgeaufw_vom_lohn_y_tu_bis_2004 + item_2 + item_3
+    out = _vorsorgeaufw_vom_lohn_y_sn_bis_2004 + item_2 + item_3
 
     return out
 
 
 @dates_active(end="2019-12-31")
-def _vorsorgeaufw_vom_lohn_y_tu_bis_2004(
-    bruttolohn_m_tu: float,
-    gemeinsam_veranlagt_tu: bool,
+def _vorsorgeaufw_vom_lohn_y_sn_bis_2004(
+    bruttolohn_m_sn: float,
+    anz_personen_sn: int,
     eink_st_abzuege_params: dict,
 ) -> float:
     """Calculate precautionary expenditures until 2019 for singles.
 
     Parameters
     ----------
-    bruttolohn_m_tu
-        See :func:`bruttolohn_m_tu`.
-    gemeinsam_veranlagt_tu
-        See :func:`gemeinsam_veranlagt_tu`.
+    bruttolohn_m_sn
+        See :func:`bruttolohn_m_sn`.
+    anz_personen_sn
+        See :func:`anz_personen_sn`.
     eink_st_abzuege_params
         See params documentation :ref:`eink_st_abzuege_params <eink_st_abzuege_params>`.
 
     Returns
     -------
 
     """
-    if gemeinsam_veranlagt_tu:
+    if anz_personen_sn > 1:
```

Comment:
Remove if, do the formula with
```
0.5 = 1 / anz_personen_sn
2 = anz_personen_sn
```
This trick might apply at a bunch of places, actually. Feel free to open an issue if you do not want to tackle it here.